addons/skin.estuary/xml/MusicVisualisation.xml

```diff
@@ -19,16 +19,16 @@
 				<fadetime>400</fadetime>
 				<animation effect="fade" start="0" end="100" time="400">WindowOpen</animation>
 				<animation effect="fade" start="100" end="0" time="300">WindowClose</animation>
-				<texture background="true" colordiffuse="88FFFFFF">$INFO[Player.Art(fanart)]</texture>
-				<visible>String.IsEmpty(Window(Visualisation).Property(ArtistSlideshow.Image))</visible>
-			</control>
+				<texture background="true">$INFO[Player.Art(fanart)]</texture>
+				<visible>!Visualisation.Enabled | [Visualisation.Enabled + String.IsEmpty(Window(Visualisation).Property(ArtistSlideshow.Image))]</visible>
+				</control>
 			<control type="image">
 				<aspectratio>scale</aspectratio>
 				<fadetime>400</fadetime>
 				<animation effect="fade" start="0" end="100" time="400">WindowOpen</animation>
 				<animation effect="fade" start="100" end="0" time="300">WindowClose</animation>
 				<texture background="true" colordiffuse="88FFFFFF">$INFO[Window(Visualisation).Property(ArtistSlideshow.Image)]</texture>
-				<visible>!String.IsEmpty(Window(Visualisation).Property(ArtistSlideshow.Image))</visible>
+				<visible>Visualisation.Enabled + !String.IsEmpty(Window(Visualisation).Property(ArtistSlideshow.Image))</visible>
```

Comment:
don't understand this one either... why only show artistslideshow images when visualisation is enabled?